pkg/features/patch_kube_features.go

```diff
@@ -13,16 +13,30 @@ var (
 	// Enables the AWS EBS CSI migration for the Attach/Detach controller (ADC) only.
 	ADCCSIMigrationAWS featuregate.Feature = "ADC_CSIMigrationAWS"
 
+	// owner: @fbertina
+	// alpha: v1.22
+	//
+	// Enables the Azure Disk CSI migration for the Attach/Detach controller (ADC) only.
+	ADCCSIMigrationAzureDisk featuregate.Feature = "ADC_CSIMigrationAzureDisk"
+
 	// owner: @jsafrane
 	// alpha: v1.21
 	//
 	// Enables the Cinder CSI migration for the Attach/Detach controller (ADC) only.
 	ADCCSIMigrationCinder featuregate.Feature = "ADC_CSIMigrationCinder"
+
+	// owner: @fbertina
+	// alpha: v1.22
```

Comment:
It says `alpha` here, but `featuregate.Beta` below. Can you please switch the doc of all gates to beta? It looks like an oversight.

Reply:
Fixed